Needed by Armadeus UDC test to know if driver is up or not.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-3.8-rc1/drivers/usb/gadget/imx_udc.c
===================================================================
--- linux-3.8-rc1.orig/drivers/usb/gadget/imx_udc.c	2013-01-01 21:19:56.000000000 +0100
+++ linux-3.8-rc1/drivers/usb/gadget/imx_udc.c	2013-01-01 21:23:03.000000000 +0100
@@ -1490,6 +1490,7 @@
 	if (ret)
 		goto fail4;
 
+	printk("%s probed\n", driver_name);
 	return 0;
 fail4:
 	for (i = 0; i < IMX_USB_NB_EP + 1; i++)
